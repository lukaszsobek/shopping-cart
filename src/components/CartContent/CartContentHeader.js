import React from "react";

const CartContentHeader = () => {
    return (
        <div className="cart-content-header">
            <div className="item-number">#</div>
            <div className="item-name">Name</div>
            <div className="item-comments">Comments</div>
            <div className="item-net-price">Price</div>
            <div className="item-tax">Tax</div>
            <div className="item-gross-price">Total Item</div>
        </div>
    )
};

export default CartContentHeader;