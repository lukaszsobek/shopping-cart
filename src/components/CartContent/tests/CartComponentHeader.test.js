import CartContentHeader from "../CartContentHeader";

describe("Renders",() => {
    it("header of cart content", () => {
        expect(CartContentHeader()).toMatchSnapshot();
    })
})