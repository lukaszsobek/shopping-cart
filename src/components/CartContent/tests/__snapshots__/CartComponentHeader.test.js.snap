// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders header of cart content 1`] = `
<div
  className="cart-content-header"
>
  <div
    className="item-number"
  >
    #
  </div>
  <div
    className="item-name"
  >
    Name
  </div>
  <div
    className="item-comments"
  >
    Comments
  </div>
  <div
    className="item-net-price"
  >
    Price
  </div>
  <div
    className="item-tax"
  >
    Tax
  </div>
  <div
    className="item-gross-price"
  >
    Total Item
  </div>
</div>
`;
